ubuntu/16.04/usr/local/share/bootstrap/common_functions.sh

```diff
@@ -235,3 +235,38 @@ function test_remote_ports() {
     timeout 1 bash -c "cat < /dev/null > /dev/tcp/${SERVICE_PARAMS[0]}/${SERVICE_PARAMS[1]}" 2>/dev/null || return 1
   done
 }
+
+function do_ownership() {
+  local OWNERSHIP_PATH=($1)
+  local USER="$2"
+  local GROUP="$3"
+  local ALLOW_GROUP_WRITE="${4:-true}"
+  if [ "$IS_CHOWN_FORBIDDEN" != 'true' ]; then
+    find "${OWNERSHIP_PATH[@]}" \( ! -user "${USER}" -or ! -group "${GROUP}" \) -exec chown "${USER}:${GROUP}" {} +
+  fi
+  do_read_write_permissions "${OWNERSHIP_PATH[@]}" "${ALLOW_GROUP_WRITE}"
```

Comment:
This doesn't pass through the paths correctly to `do_read_write_permissions`, only the first one.